src/pages/components/menu.mdx

```diff
@@ -48,102 +47,125 @@ components:
 
 ### Default
 
-Works in combination with a Trigger to display a list of menu items.
+Display a list of menu items. Uses a default Garden [Button](/components/button) internally.
 
-import MenuDefault from '../../examples/dropdowns/menu/Default.tsx';
-import MenuDefaultCode from '!!raw-loader!../../examples/dropdowns/menu/Default.tsx';
+import Default from '../../examples/menu/Default.tsx';
+import DefaultCode from '!!raw-loader!../../examples/menu/Default.tsx';
 
-<CodeExample code={MenuDefaultCode}>
-  <MenuDefault />
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Button
+
+Configure a custom menu trigger.
+
+import ButtonExample from '../../examples/menu/Button.tsx';
+import ButtonCode from '!!raw-loader!../../examples/menu/Button.tsx';
+
+<CodeExample code={ButtonCode}>
+  <ButtonExample />
 </CodeExample>
 
 ### Arrow
 
 An arrow can be added to the Menu to give the user a sense of what element is triggering it.
 
-import MenuArrow from '../../examples/dropdowns/menu/Arrow.tsx';
-import MenuArrowCode from '!!raw-loader!../../examples/dropdowns/menu/Arrow.tsx';
+import Arrow from '../../examples/menu/Arrow.tsx';
+import ArrowCode from '!!raw-loader!../../examples/menu/Arrow.tsx';
 
-<CodeExample code={MenuArrowCode}>
-  <MenuArrow />
+<CodeExample code={ArrowCode}>
+  <Arrow />
 </CodeExample>
 
 ### Danger
 
 Use danger styling for Menu items that enable destructive action.
 
-import MenuDanger from '../../examples/dropdowns/menu/Danger.tsx';
-import MenuDangerCode from '!!raw-loader!../../examples/dropdowns/menu/Danger.tsx';
+import Danger from '../../examples/menu/Danger.tsx';
+import DangerCode from '!!raw-loader!../../examples/menu/Danger.tsx';
 
-<CodeExample code={MenuDangerCode}>
-  <MenuDanger />
+<CodeExample code={DangerCode}>
+  <Danger />
 </CodeExample>
 
 ### Disabled
 
 Items within a Menu can be disabled to prevent a user from interacting with them.
 
-import MenuDisabled from '../../examples/dropdowns/menu/Disabled.tsx';
-import MenuDisabledCode from '!!raw-loader!../../examples/dropdowns/menu/Disabled.tsx';
+import Disabled from '../../examples/menu/Disabled.tsx';
+import DisabledCode from '!!raw-loader!../../examples/menu/Disabled.tsx';
 
-<CodeExample code={MenuDisabledCode}>
-  <MenuDisabled />
+<CodeExample code={DisabledCode}>
+  <Disabled />
 </CodeExample>
 
 ### Media items
 
 Media elements add even more context to a Menu.
 
-import MenuMedia from '../../examples/dropdowns/menu/Media.tsx';
-import MenuMediaCode from '!!raw-loader!../../examples/dropdowns/menu/Media.tsx';
+import Media from '../../examples/menu/Media.tsx';
+import MediaCode from '!!raw-loader!../../examples/menu/Media.tsx';
 
-<CodeExample code={MenuMediaCode}>
-  <MenuMedia />
+<CodeExample code={MediaCode}>
+  <Media />
 </CodeExample>
 
-### Meta
+### Meta items
```

Comment:
Could we make this "Meta text"? So it's conceptually consistent throughout the document.

Reply:
Thanks, pushed up that change.